lms/lmstests/public/linters/tasks.py

```diff
@@ -12,10 +14,15 @@
 
 @app.task
 def run_linter_on_solution(solution_pk: str) -> None:
+    _logger.info('Start running check solution %s', solution_pk)
+    checker = services.LinterChecker(solution_pk, _logger)
+
     try:
-        _logger.info('Start running check solution %s', solution_pk)
-        checker = services.LinterChecker(solution_pk, _logger)
         checker.initialize()
+    except models.Solution.DoesNotExist:
+        _logger.exception('Solution %s does not exist', solution_pk)
+
+    try:
         checker.run_check()
-    except Exception:
+    except (CeleryError.TaskError, OperationalError):
```

Comment:
Are you sure it's not just celery.exceptions.TaskError?
https://docs.celeryproject.org/en/stable/reference/celery.exceptions.html#celery.exceptions.TaskError

Reply:
'cause `OperationalError` is not under `TaskError`
It basically should catch connection errors

Reply:
Now I got what you meant XD Yeah, you're right, fixing it. Great catch